parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -37,15 +37,16 @@ installDeps() {
     BLOBFUSE2_VERSION="2.2.1"
 
     # keep legacy version on ubuntu 16.04 and 18.04
-    if [ "${OSVERSION}" == "16.04" ]; then
-        BLOBFUSE_VERSION="1.3.7"
-    fi
     if [ "${OSVERSION}" == "18.04" ]; then
         BLOBFUSE2_VERSION="2.2.0"
     fi
 
-    pkg_list+=(blobfuse2=${BLOBFUSE2_VERSION})
-    if [[ $(isARM64) != 1 ]]; then
+    # Currently skipping blobfuse install on 24.04 since packages are not available upstream, remove this once they are
+    if [[ "${OSVERSION}" != "24.04" ]]; then
```

Comment:
I remember there was a 2.3.0~preview.1 version for blobfuse2, can we not use that?

Reply:
will take it up in a separate PR once I confirm with Andy and CSI team that its the correct version to use for 24.04